package.json

```diff
@@ -65,7 +65,7 @@
     "protobufjs": "6.8.8",
     "sinon": "7.3.1",
     "ts-node": "8.0.3",
-    "typedoc": "0.14.2",
+    "typedoc": "^0.15.0",
```

Comment:
Why are you upgrading this?

In any case, please stick to exact version numbers (`npm i -DE typedoc@0.15.0`) to ensure a completely reproducible build, even in the presence of unwanted semver violations.

Reply:
I must have made a mistake and push the patch incorrectly.

Reply:
Change made now